editor/js/Loader.js

```diff
@@ -995,10 +995,10 @@ function Loader( editor ) {
 		const { MeshoptDecoder } = await import( 'three/addons/libs/meshopt_decoder.module.js' );
 
 		const dracoLoader = new DRACOLoader();
-		dracoLoader.setDecoderPath( '../examples/jsm/libs/draco/gltf/' );
+		dracoLoader.setDecoderPath( 'three/addons/libs/draco/gltf/' );
 
 		const ktx2Loader = new KTX2Loader();
-		ktx2Loader.setTranscoderPath( '../examples/jsm/libs/basis/' );
+		ktx2Loader.setTranscoderPath( 'three/addons/libs/basis/' );
```

Comment:
This change does not work. You can only use the `addons` alias if your import via ES6 import syntax. If you load from a URL e.g. via fetch, you have to use the real directoy path meaning `examples/jsm`.

Reply:
Sorry, I again checked all the changes and fixed lines that do not work. 
It's my bad that I only test a subset and I'll do more careful tests before my next prs. Thanks!